setup.py

```diff
@@ -37,20 +25,15 @@ def get_decoder_pkg_url(version, artifacts_root=None):
     if plat == 'windows':
         plat = 'win'
 
-    is_ucs2 = sys.maxunicode < 0x10ffff
-    m_or_mu = 'mu' if is_ucs2 else 'm'
-
     # ABI does not contain m / mu anymore after Python 3.8
     if sys.version_info.major == 3 and sys.version_info.minor >= 8:
         m_or_mu = ''
+    else:
+        is_ucs2 = sys.maxunicode < 0x10ffff
+        m_or_mu = 'mu' if is_ucs2 else 'm'
 
     pyver = ''.join(str(i) for i in sys.version_info[0:2])
 
-    if not artifacts_root:
-        artifacts_root = 'https://community-tc.services.mozilla.com/api/index/v1/task/project.deepspeech.deepspeech.native_client.{branch_name}.{tc_arch_string}/artifacts/public'.format(
-            branch_name=branch,
-            tc_arch_string=tc_arch)
-
     return 'ds_ctcdecoder @ {artifacts_root}/ds_ctcdecoder-{ds_version}-cp{pyver}-cp{pyver}{m_or_mu}-{platform}_{arch}.whl'.format(
```

Comment:
do we want to keep the name like that?

Reply:
like what?

Reply:
something that does not include "ds" maybe :)